site/docs/getting-started/sso-setup.md

```diff
@@ -0,0 +1,73 @@
+
+# Log into Estuary Using SSO
+
+You can register or log into the [Estuary dashboard](https://dashboard.estuary.dev/) with one of several methods. Sign in with either:
+* A Google account
+* A GitHub account
+* An Azure account
+* SSO (Single Sign-On)
```

Comment:
Thinking about this for the first time... we might want to highlight that this is SAML ... since the other ones are also SSO.

@dyaffe - any thoughts on this?